src/lolremez.cpp.bak

```diff
@@ -0,0 +1,334 @@
+//
```

Comment:
Does this backup file get used anywhere? I think we'll be able to refer to the git history if we need to understand the XMOS specific changes that have been made?